site/contents/write_ups/2022/04-05-22_writeup.md

```diff
@@ -0,0 +1,97 @@
+
+Often times a data science's job is not to evaluate, but to make something look the best based. Data science is a lot about representing information. Whilst part of that is using that representation to best understand it from your point of view, an equally important part is to represent the data in a way that says the message you (or the client) want. When you're trying to get a product out, if you have a vested interest, it makes it very hard to stay objective.
+
+People see models as not harmful by default. Many people throw their data into a black box which they don't understand. Data scientists need to realise that there are consequences to the work that they do. Unless people are economically punished for unethical practises, where is the inscentive for them to stop?
+
+## Are interventions/research that serve capitalist interests always bad/harmful?
+
+What are our assumptions about what is good/bad? Is working not-for-profit automatically bad? What are we aiming for: in terms of interventions (e.g. regulation) or in terms of values (e.g. transparency).
+
+Ethical harms are less tangible sometimes than other regulated things. Is it possible to circumvent huge scandals? Or is it necessary? Even in more critical tangible systems (planes/cars), things keep on until scandal because it's deniable that there's a problem. What is the structure that will catch these problems earlier on? Negative feedback loops come to mind. Why do organisations bring in standards? Because employees asked for it.
+
+The aviation industry provides an interesting case study: plane crashes cause mistrust and low sales, is there a feedback loop in the same way? FAA (aviation industry) has very heavy-handed regulation, but still has scandals and failures. But maybe there's another world where we just accept planes fall out of the sky sometimes? Big tech self regulation...is that really a good idea?
+
+There appears to be a presumption 100% transparency is good. Even if you're working in not for profit, you're still not going to be 100% transparent at all times. Why are we asking for transparency? Transparency is in some sense the opposite to trust but a direct call for transparency is not the same as truth.
+
+Why are we not transparent? Coorporate interests are the motivation in private companies but what about public sector? Competitive interests might be apparent, but also privacy and potentially national security measures. On a personal level, one might not be confident in own ability as developer or not willing to share. What would we want to be transparent about? Transparency to the end user: which variables are impacting you? Transparency to the regulator: which variables are causing this group of people, or this selection of (e.g.) failed loan applications to fail?
```

Comment:
```suggestion
Might we have good reasons for not being transparent? Coorporate interests are the motivation in private companies but what about public sector? Competitive interests might be apparent, but there are likely to be "good" reasons relating to privacy and national security. On a personal level, sometimes people are not confident in their own ability and so not willing to share. What would we want to be transparent about? This might look different to different audiences (Transparency to the end user: which variables are impacting you? Transparency to the regulator: which variables are causing this selection of loan applications to fail?).
```